OrbitGl/CaptureWindow.h

```diff
@@ -63,6 +63,8 @@ class CaptureWindow : public GlCanvas {
   void ToggleDrawHelp();
   void set_draw_help(bool draw_help);
   [[nodiscard]] TimeGraph* GetTimeGraph() { return &time_graph_; }
+  [[nodiscard]] const CaptureData* GetCaptureData() const { return time_graph_.GetCaptureData(); }
```

Comment:
Nit: Is this (and the setter below) needed? As far as I can see, the capture data is accessed through time_graph in all cases. It could be clearer to remove both of them and explicitely use `CaptureWindow::GetTimeGraph()->Get/SetCaptureData` where it's needed.

Reasoning: From the outside I find it is not always clear if these methods are proxies for convenience, or if `CaptureWindow::time_graph_` and `CaptureWindow` could hold different instances of `CaptureData`. Since we're exposing the TimeGraph anyways, I don't see the reason to do this.

Reply:
I agree, I removed it, thanks!